targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/device/TOOLCHAIN_GCC_ARM/NRF52840.ld

```diff
@@ -35,7 +35,7 @@
 #endif
 
 #define MBED_RAM0_START MBED_RAM_START
-#define MBED_RAM0_SIZE  0xF8
+#define MBED_RAM0_SIZE  0x100
```

Comment:
Why is the delta for the NRF52840 larger than the NRF52832?

Were the sizes simply incorrect?

Reply:
The +4 for the NRF52832 is an off-by-one error I believe.

The +8 for the NRF52840 is the SPI3 being moved down and extending the vector table by 2 slots.